doc/howto/storage_volumes.md

```diff
@@ -37,6 +40,14 @@ To show state information about a specific volume, use the following command:
 
 In both commands, the default {ref}`storage volume type <storage-volume-types>` is `custom`, so you can leave out the `custom/` when displaying information about a custom storage volume.
 
+````
+```` {group-tab} UI
+
+To view available storage volumes in the UI, from the {guilabel}`Storage` section of the main navigation, select {guilabel}`Volumes`..
```

Comment:
The cli section explains how to filter volumes by pool, the UI can do that as well, might be worth a note.